package.json

```diff
@@ -2,30 +2,39 @@
   "name": "rubiks-cube",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "test:nowatch": "react-scripts test --watchAll=false",
```

Comment:
Rather than set CI=true to stop tests from running in "watch mode", we can simply pass this arg.